hostedWidgetCustomization/README.md

```diff
@@ -0,0 +1,77 @@
+# Hosted Widget Customizations
+
+Additional ways to configure the Amazon Connect Hosted Widget on your website and further personalize the branding.
+
+View the following sample code for customizing the widget. Follow the ["Admin Guide: Add chat to your website"](https://docs.aws.amazon.com/connect/latest/adminguide/add-chat-to-website.html) to get started, and replace the example snippets with your generated `<script>` code.
+
+```html
+<!-- EXAMPLE SNIPPET - Edit all "<REPLACE_ME>" values -->
+<script type="text/javascript">
+  (function (w, d, x, id) {
+    s = d.createElement("script");
+    s.src =
+      "https://<REPLACE_ME>.cloudfront.net/amazon-connect-chat-interface-client.js";
+    s.async = 1;
+    s.id = id;
+    d.getElementsByTagName("head")[0].appendChild(s);
+    w[x] =
+      w[x] ||
+      function () {
+        (w[x].ac = w[x].ac || []).push(arguments);
+      };
+  })(window, document, "amazon_connect", "<REPLACE_ME>");
+  amazon_connect("styles", {
+    openChat: { color: "#ffffff", backgroundColor: "#07b62a" },
+    closeChat: { color: "#ffffff", backgroundColor: "#07b62a" },
+  });
+  amazon_connect("snippetId", "<REPLACE_ME>");
+  amazon_connect("supportedMessagingContentTypes", [
```

Comment:
do we want to also add a note about rich messaging enabled vs disabled? Or just in general call out that other methods can be added here to pass display name, contact attributes, etc and add link?

Reply:
Yes we can add follow up documentation for rich messaging (the Admin guide doesn't really explain that either) - however, customer cannot pass displayName/contact attributes in this snippet. 

If they build a custom widget, they can of course: https://github.com/amazon-connect/amazon-connect-chat-interface/blob/master/DOCUMENTATION.md#chat-interface-configuration

Reply:
why can't they pass display name/contact attributes? is that specific to if they specify this auto launch behavior?